### ISSUE TITLE
[DOCUMENTATION]: Mega Menu - Component MUST render in our documentation site to be evaluated properly

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#10426]
# [508-defect-1 :exclamation: Launchblocker](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-1)

**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

Our [Mega Menu component](https://department-of-veterans-affairs.github.io/veteran-facing-services-tools/visual-design/components/megamenu/) is not rendering on the Veteran Facing Services Tools documentation site. This is preventing accessibility experts from testing the component, and creates risks for incorrect use or accessibility issues.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## Acceptance Criteria

- [ ] Component renders properly on the documentation page
- [ ] Component has no axe browser plugin violations

## Screenshots or Trace Logs

<img width="1064" alt="Screen Shot 2020-06-18 at 9 57 50 AM" src="https://user-images.githubusercontent.com/934879/85037337-d6f07300-b14a-11ea-9b54-022489ec180f.png">

**Comment by 1Copenut** [comment]
Issue moved to [department-of-veterans-affairs/vets-design-system-documentation #223](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/223) via [**ZenHub**](https://www.zenhub.com/)
